src/constructs/aws/Queue.ts

```diff
@@ -286,14 +291,22 @@ export class Queue extends AwsConstruct {
         };
     }
 
+    private getBatchSize(): number {
+        return this.configuration.batchSize ?? 1;
+    }
+
     private getMaximumBatchingWindow(): number {
         return this.configuration.maxBatchingWindow ?? 0;
     }
 
+    private getMaximumConcurrency(): number {
+        return this.configuration.maxConcurrency ?? 2;
+    }
+
     private appendFunctions(): void {
-        // The default batch size is 1
-        const batchSize = this.configuration.batchSize ?? 1;
```

Comment:
I don't think it's worth adding a method for this simple line.

Reply:
ok, I will revert it, as i intended to make it consistent with other fields.